src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/TelemetryClientService.kt

```diff
@@ -75,6 +75,7 @@ class TelemetryClientService(
       "visitStart" to formatDateTimeToString(visit.startTimestamp),
       "visitStatus" to visit.visitStatus.name,
       "applicationReference" to visit.applicationReference,
+      "hasContactInformation" to ((visit.visitContact != null).toString()),
```

Comment:
adding this for all scenarios - booked / updated / cancelled in case we want to count how many messages are sent